wss/1.0/modules/ROOT/pages/wss-examples.adoc

```diff
@@ -3,7 +3,29 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+The following WSS module examples show you how to validate a Web Service Security (WSS) configuration in your incoming SOAP requests:
 
+* xref:validate-wss-examples.adoc[Validate Operation Examples]
+** xref:validate-wss-examples.adoc#decrypt[Decrypt Message Example]
+** xref:validate-wss-examples.adoc#usernameToken[Validate UsernameToken Examples]
+*** xref:validate-wss-examples.adoc#ldap-validate[LDAP Authentication]
+*** xref:validate-wss-examples.adoc#username-password[Credentials Authentication]
+** xref:validate-wss-examples.adoc#signature[Validate Signature Examples]
+*** xref:validate-wss-examples.adoc#validate-trustore-signature[Trustore]
+*** xref:validate-wss-examples.adoc#validate-binary-security-token[Binary Security Token]
+*** xref:validate-wss-examples.adoc#validate-x509-certificate[X.509 Certificate]
+** xref:validate-wss-examples.adoc#timestamp[Validate Timestamp Example]
+** xref:validate-wss-examples.adoc#saml[Validate SAML Examples]
+*** xref:validate-wss-examples.adoc#signed-saml[Signed SAML]
+*** xref:validate-wss-examples.adoc#unsigned-saml[Unsigned SAML]
+
+
+The following WSS module examples show you how to apply a Web Service Security (WSS) configuration to your outgoing SOAP responses:
```

Comment:
not bold

Reply:
It's not bold. There's something with the highlight syntax in this view.